integration-tests/run-intg-test.py

```diff
@@ -405,25 +424,26 @@ def build_module(module_path):
     """
     logger.info('Start building a module. Module: ' + str(module_path))
     if sys.platform.startswith('win'):
-        subprocess.call(['mvn', 'clean', 'install', '-B',
+        subprocess.call(['mvn', 'clean', 'install', '-fae', '-B',
                          '-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'],
                         shell=True, cwd=module_path)
     else:
-        subprocess.call(['mvn', 'clean', 'install', '-B',
+        subprocess.call(['mvn', 'clean', 'install', '-fae', '-B',
                          '-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'],
                         cwd=module_path)
     logger.info('Module build is completed. Module: ' + str(module_path))
 
 
 def save_log_files():
-    log_storage = Path(workspace + "/" + LOG_STORAGE)
-    if not Path.exists(log_storage):
-        Path(log_storage).mkdir(parents=True, exist_ok=True)
-    log_file_paths = LOG_FILE_PATHS[product_id]
-    if log_file_paths:
-        for file in log_file_paths:
+    log_folder = Path(workspace + "/" + LOG_STORAGE)
```

Comment:
Shall we move this to a different function other than `save_log_files()`?  AFAIU, this is used to save the multiple surefire-reports and it would be better if the function name reflects the purpose of it.

Reply:
Fixed. Please refer latest commits